content/c4-router/diagram-c4.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+!include C4_Container.puml
```

Comment:
Why to not integrate the full url : !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/master/C4_Container.puml
 ?